aeron-agent/src/main/java/io/aeron/agent/ArchiveEventLogger.java

```diff
@@ -45,6 +45,11 @@ public final class ArchiveEventLogger
 
     static final EnumSet<ArchiveEventCode> CONTROL_REQUEST_EVENTS = complementOf(of(
         CMD_OUT_RESPONSE,
+        REPLICATION_SESSION_STARTED,
+        REPLAY_SESSION_STARTED,
+        RECORDING_SESSION_STARTED,
+        REPLAY_SESSION_STATE_CHANGE,
+        RECORDING_SESSION_STATE_CHANGE,
```

Comment:
`*_SESSION_STARTED` are not needed since they are covered by existing events. See below.